net/Socket.cpp

```diff
@@ -1543,7 +1651,7 @@ bool StreamSocket::compactChunks(MessageMap& map)
 bool StreamSocket::sniffSSL() const
 {
     // Only sniffing the first bytes of a socket.
-    if (_bytesSent > 0 || _bytesRecvd != _inBuffer.size() || _bytesRecvd < 6)
+    if (getBytesSent() > 0 || getBytesRcvd() != _inBuffer.size() || getBytesRcvd() < 6)
```

Comment:
The accessor-ization is something that should be done in a separate commit - if we want that. Its far from clear to me that it helps readability - prwt. performance _bytesRecvd' I can compile in my head, but an accessor could do anything and lives somewhere else.

Also - it seems we have publicised lots of our previously private members by adding accessors to access them all; is that necessary? not a huge fan of puncturing encapsulation that keeps implementation details visible & local. We can of course make the accessors private too - but then - what value did we add here at what cost ?

Reply:
only here since these stats moved up to the parent class Socket and I wanted to clarify mutable access, i.e. ease review and having those fields private. Yes, the verb 'get' implies a most simple thing .. in theory. One likes to drop the verb for simple values (like here .. me too).

The discussion of redoing the commits (splitting etc) .. let's do that separately. 
First I like to finish the overall picture / feature .. then we see and balance for the final version.
Me still hacking on things here.

Reply:
Fair enough - if you insist on the accessors - lets make sure they have the right scope - protected if necessary; ideally private of course.

Reply:
changed to simple noun name (accessor) and moved writer to protected in revised commit.